### ISSUE TITLE
I cannot find module 'Models'.  Module 'Main' is trying to import it.

### ISSUE THREAD

**Issue by SebQuenet** [mmenestret/elm-card-drawer#1]
Given :  elm 0.18.0, 

When : run command line as explained : 
[master] 22:56:45-$ elm-make index.elm --output index.html

Then : Got the following error : 
I cannot find module 'Models'.

Module 'Main' is trying to import it.

Potential problems could be:
  * Misspelled the module name
  * Need to add a source directory or new dependency to elm-package.json

=======

After checked elm-package.json, seems I have the proper source-directories : 
    "source-directories": [
        ".",
        "models",
        "utils"
    ],

any hint ?

**Comment by mmenestret** [comment]
No, I just tried again to clone the project again, follow the instructions and it compiles... I changed the directories organization a bit (now you have to make the src/index.elm) but it still works for me.
Can you try again ?
